### PR TITLE
Make the "ask for help" button use new subforum

### PR DESCRIPTION
**Pull request by QuincyLarson**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

### CODE REVIEW

File: client/utils/challengeTypes.js [hunk 1]
```diff
@@ -70,6 +70,6 @@ exports.submitTypes = {
 exports.helpCategory = {
   [html]: 'HTML-CSS',
   [js]: 'JavaScript',
```
Comment (RandellDawson) [inline client/utils/challengeTypes.js]: @QuincyLarson Will these automatically apply the category `Curriculum Help` and the applicable subcategory?

**Comment by RandellDawson** [comment]
After some testing, I noticed challenges of `challengeType 2` like the [Basic Node and Express - Get Data from POST Requests](https://www.freecodecamp.org/learn/apis-and-microservices/basic-node-and-express/get-data-from-post-requests) should not use a topic category/subcategory of `Curriculum Help / Certification Projects`.  Instead, I think these should be `Curriculum Help / JavaScript`.

Same problem exists with `challengeType 5` challenges like [Basic Algorithm Scripting: Mutations](https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-algorithm-scripting/mutations) and with `challengeType 6` challenges like [jQuery: Target HTML Elements with Selectors Using jQuery]((https://www.freecodecamp.org/learn/front-end-libraries/jquery/target-html-elements-with-selectors-using-jquery).  They should not use a topic category/subcategory of `Curriculum Help / Certification Projects`.  Instead, they should be `Curriculum Help / JavaScript`.

**Comment by raisedadead** [comment]
Yes, I had anticipated that and have marked the PR blocked for now. Will take a look.

**Comment by raisedadead** [comment]
This needs some additional updates. Will open a fresh PR.

**Pull request closed without merging**